src/pages/Products/SelectFilter.js

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useRef, useState } from 'react';
+import css from './SelectFilter.module.scss';
+
+function SelectFilter(props) {
+  const [appear, setAppear] = useState(false);
+
+  const clickDropList = e => {
+    setAppear(!appear);
+  };
+
+  const outsideRef = useRef(null);
+
+  useEffect(() => {
+    function handleClickOutside(event) {
+      if (outsideRef.current && !outsideRef.current.contains(event.target)) {
+        setAppear(appear);
+      }
+    }
+    document.addEventListener('click', handleClickOutside);
+
+    return () => {
+      document.removeEventListener('click', handleClickOutside);
+    };
+  }, [outsideRef]);
+
+  const [selectFilter, setSelectFilter] = useState(false);
+
+  const clickChangeFilter = e => {
+    setSelectFilter(!selectFilter);
+  };
+
+  return (
+    <div className={css.select_filter} ref={outsideRef}>
+      <div
+        className={[css.drop_list, css.drop_list_title].join(' ')}
+        onClick={clickDropList}
+      >
+        추천순
+        <span
+          className={appear ? css.drop_list_arrow_up : css.drop_list_arrow_down}
+        ></span>
+      </div>
+      <ul
+        className={css.select_filter_drop}
+        style={{ display: appear ? 'block' : 'none' }}
+      >
+        {props.option.map(option => (
```

Comment:
eslint 설정에 state, prop 등에 구조분해 할당을 적용하라는 규칙이 있어서
경고가 발생할 것 같아요!
```suggestion
        const { option } = props;
        ...
        {option.map(option => (
```
이런 식으로 수정하시면 경고가 사라질 것 같습니다!

Reply:
네~!! 수정하겠습니다~ 리뷰 감사합니다~~ 👍👍👍